public/index.tsx

```diff
@@ -73,15 +73,39 @@ function parseQueryString(): Params {
 }
 const params = parseQueryString();
 
-const args = {
-  //baseurl: "http://dev-aics-dtp-001.corp.alleninstitute.org/cellviewer-1-4-0/Cell-Viewer_Thumbnails/",
-  baseurl: "https://s3-us-west-2.amazonaws.com/bisque.allencell.org/v1.4.0/Cell-Viewer_Thumbnails/",
-  cellid: 2025,
-  cellPath: "AICS-22/AICS-22_8319_2025_atlas.json",
-  fovPath: "AICS-22/AICS-22_8319_atlas.json",
-  fovDownloadHref: "https://files.allencell.org/api/2.0/file/download?collection=cellviewer-1-4/?id=F8319",
-  cellDownloadHref: "https://files.allencell.org/api/2.0/file/download?collection=cellviewer-1-4/?id=C2025",
-  initialChannelSettings: VIEWER_3D_SETTINGS,
+const decodeUrl = (url: string): string => {
+  const decodedUrl = decodeURIComponent(url);
+  return decodedUrl.endsWith("/") ? decodedUrl.slice(0, -1) : decodedUrl;
+};
+
+/** Tries to parse a string as a list of URLs */
+const tryDecodeURLList = (url: string, delim = ","): string[] | undefined => {
+  if (!url.includes(delim)) {
+    return undefined;
```

Comment:
Nit: Just a thought - rather than returning `undefined` in cases of invalid input, might we consider returning an empty array instead? This would make the behavior more predictable and debugging more straightforward.

Reply:
The idea of this function is to either parse a list of two or more URLs out of a `string`, or to determine that the `string` is not a well-formatted URL list (likely because it's just a single URL, though maybe because it's incorrectly formatted). Returning `undefined` in that second case lets me concisely fall back to parsing a single URL below, on line 209:

`const imageUrls = tryDecodeURLList(params.url) ?? decodeUrl(params.url);`

Basically, "get a URL list out of this string, or if you can't do that, format out a single URL." I've updated the doc comment to be a bit more specific about the function's purpose.

Reply:
For the sake of discussion, the function could also return `[url]` if there's no comma in it! (Hint: I have already approved this pr)